asv_bench/benchmarks/utils.py

```diff
@@ -67,27 +67,21 @@
 BINARY_OP_DATA_SIZE = {
     "big": [
         ((5000, 5000), (5000, 5000)),
-        # the case extremely inefficient
-        # ((20, 500_000), (10, 1_000_000)),
         ((500_000, 20), (1_000_000, 10)),
     ],
     "small": [
         ((250, 250), (250, 250)),
-        ((20, 10_000), (10, 25_000)),
         ((10_000, 20), (25_000, 10)),
```

Comment:
why do you touch this (seemingly not OmniSci-related) part in OmniSci-related PR?

Reply:
removed